packages/docs/src/pages/en/blog/building-a-basic-nuxt-application-with-vuetify.md

```diff
@@ -0,0 +1,565 @@
+
+- Each list item is a v-list-item component, itself composed of a title (v-list-item-title component).
+- Each title is represented by an icon (v-icon component) and a text.
+
+The appearance of the menu is controlled by a reactive variable, here named drawer. If it is set to false (default value), the menu is not displayed. As soon as it changes to true, the menu appears. The change from false to true (or vice versa) happens when the mdi-apps button in the toolbar is clicked.
+
+The menu also needs to be linked to this reactive variable. To achieve this, Vuetify uses the v-model  attribute in the v-navigation-drawer  com- ponent. Many Vuetify components use the v-model attribute to link the component to a reactive variable in the program.
+
+Clicking on each navigation menu item is handled by the @click attribute, which is a shorthand for v-on:click  provided by Vue.js. We call the navigateTo(page) method, which displays the page specified as a para- meter, such as "/"  (to display index.vue), "/products"  (to display products.vue), or "/contact" (to display the contact.vue page).
+
+Let's verify the proper functioning of this mechanism by displaying the products.vue page. We assume that each page of the application has been created similarly to the index.vue page. These pages will be created in the next section if needed.
+
+![](https://cdn.vuetifyjs.com/docs/images/blog/building-a-basic-nuxt-application-with-vuetify/12.png){ height=500 }
+
+We can now navigate from page to page within the application.
+
+## Creating the other pages of the application
+
+The products.vue page, displayed previously, must first be created. We create it using the same principle as the index.vue page that was created earlier.
+
+Here is the code describing each of the components associated with the pages of the application:
+
+```html { resource="pages/products.vue" }
+<template>
+  <v-container class="fill-height d-flex flex-column align-center justify-center text-center">
+    <v-icon size="64">mdi-cube-outline</v-icon>
+
+    <div class="mt-3">MyApplication's Products Page</div>
+  </v-container>
+</template>
+```
+
+## Adding buttons to the toolbar
+
+Finally, we might want to insert new buttons into the toolbar at the top of the screen, on the right side. These buttons can provide direct access to one of the application’s pages, without using the previously implemented naviga- tion menu.
+
+We therefore insert buttons in the form of icons into the application's tool- bar. The app.vue component becomes:
+
+```html
+<template>
+  <v-app>
+    <!-- Toolbar at the top of the screen -->
+    <v-app-bar app color="primary" height="48">
+      <!-- Title of the application with adjusted margin and font size -->
+      <v-toolbar-title class="text-h6 ml-3">
+      <v-icon @click="drawer = !drawer">mdi-apps</v-icon>
```

Comment:
`<v-icon @click="drawer = !drawer">mdi-apps</v-icon> ` to `<v-icon @click="drawer = !drawer" icon="mdi-apps" />`